fairscale/nn/pipe/messages.py

```diff
@@ -0,0 +1,137 @@
+import pickle
+from queue import Empty as QueueEmpty
+from queue import Queue
+from typing import Any, List
+
+import numpy as np
+import torch
+from torch import Tensor
+
+from fairscale.nn.model_parallel import get_pipeline_parallel_group
+
+from .types import MESSAGE_GENERATION_START, InputDevice, PipeMessage, Tensors, TransportConfig
+
+# FIXME Why is 256 ok for training but not for tests?
+MESSAGE_TENSOR_SIZE = 1024  # 256
+
+MessageQueues: List[Queue] = [Queue() for _ in range(MESSAGE_GENERATION_START)]
+
+
+def pyobject_to_tensor(obj: Any) -> Tensor:
```

Comment:
We have similar code in fairscale/optim/utils.py. Maybe we should create a fairscale.utils package to avoid duplication.

Reply:
oh I didn't know about that, created fairscale/utils/object.py